ynput/OpenPype
# Maya: Implement image file node loader

## Brief description

Implements a loader for loading texture image into a `file` node in Maya.

## Description

Similar to Maya's hypershade creation of textures on load you have the option to choose for three modes of creating:
- Texture
- Projection
- Stencil

These should match what Maya generates if you create those in Maya.

- [x] Load and manage file nodes
- [x] Apply color spaces after #4195 
- [x] Support for _either_ UDIM or image sequence - currently it seems to always load sequences as UDIM automatically.
    - [ ] Add support for animation sequences of UDIM textures using the `<f>.<udim>.exr` path format?

## Additional info

This was implemented during the work on #4283 and tested against image output publishes.
Here's an example video where at the end I'm loading the images into maya for shading:

https://user-images.githubusercontent.com/2439881/212285701-c5351ba8-35cb-4015-9118-6552837ff5d6.mp4

## Documentation (add _"type: documentation"_ label)

Currently none. Should there be?

## Testing notes:

1. Make sure you have a published image (or udim sequence) with family "image", "plate" or "render" and an "exr", "tif", "png" or "jpg" representation.
2. Load away!
